src/renderer/pages/Onboarding/WalletConnect/ManageStep/ManageStep.tsx

```diff
@@ -0,0 +1,197 @@
+import cn from 'classnames';
+import { useEffect, useState } from 'react';
+import { Controller, useForm, SubmitHandler } from 'react-hook-form';
+
+import { useI18n, useStatusContext } from '@renderer/app/providers';
+import {
+  AccountId,
+  Chain,
+  ChainType,
+  ErrorType,
+  NoID,
+  SigningType,
+  WalletType,
+  AccountType,
+  WalletConnectAccount,
+} from '@renderer/shared/core';
+import { Button, Input, InputHint, HeaderTitleText, SmallTitleText, Icon } from '@renderer/shared/ui';
+import { toAccountId } from '@renderer/shared/lib/utils';
+import { MultiAccountList } from '@renderer/entities/account/ui/MultiAccountsList/MultiAccountsList';
+import { chainsService } from '@renderer/entities/network';
+import { IconNames } from '@renderer/shared/ui/Icon/data';
+import { walletModel } from '@renderer/entities/wallet';
+
+type WalletForm = {
+  walletName: string;
+};
+
+type WalletTypeName = WalletType.NOVA_WALLET | WalletType.WALLET_CONNECT;
+
+type Props = {
+  accounts: string[];
+  pairingTopic: string;
+  sessionTopic: string;
+  type: WalletTypeName;
+  onBack: () => void;
+  onComplete: () => void;
+};
+
+const ManageStep = ({ accounts, type, pairingTopic, sessionTopic, onBack, onComplete }: Props) => {
```

Comment:
Why it's not fully controlled by effector model? It's a perfect candidate
By the way if `StatusContext` were made with effector, it could have been used inside every model not only in UI
And submit handler would have looked like this:
```javascript
const submitForm = (event: FormEvent) => {
    event.preventDefault();
    submit();
  };
```
Right now it kind of works but over all it's another legacy code
Fully working effector examples are _CreateContactForm_, _CurrencyForm_ (with few mistakes because of i18n not moved to effector yet)
Don't think we have time to rewrite it to effector in this sprint, so please add a todo 🙏